spec/shr_oncologycore.txt

```diff
@@ -42,151 +42,167 @@ Concept:        LNC#21858-6 "Grade Cancer"
 Concept:        SCT#246453008 "Course of illness (attribute)"
                 FindingResult from CancerProgressionVS
                 FindingTopicCode is SCT#246453008 "Course of illness (attribute)"
                 SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
-0..0			AnatomicalLocation
-0..0            Device
 // CancerProgressionEvidence is not yet part of M-CODE, but I don't see a better way to include it for FluxNotes
-0..*            CancerProgressionEvidence		
+0..*            CancerProgressionEvidence
+0..0			AnatomicalLocation
+0..0            Device	
 
 		Element:		CancerProgressionEvidence
 		Description:	"The type of evidence considered in determining disease progression."
 		Value:			CodeableConcept from CancerProgressionEvidenceVS	
 
 
-EntryElement:   CancerStageInformation
+Abstract Element:   CancerStagePanel
 Based on:       Observation
-Description:    "The extent a cancer has spread in the body (source: NCI Dictionary of Cancer Terms)"
+Description:    "Cancer staging is a way of describing the size of a cancer and whether it has spread into nearby lymph nodes or to other parts of the body. This abstract class is the parent of panels containing information related to clinical and pathologic staging."
 Concept:        SCT#258232002 "Cancer staging (tumor staging)"
-                FindingTopicCode is SCT#258232002 "Cancer staging (tumor staging)"
-    0..0        FindingResult
-    0..1        StageTiming  // not a component since it is not a finding, but a further description of the circumstances of the observation
+                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
+    0..1        FindingMethod from TNMTumorStagingSystemVS if covered
+    0..0        FindingResult  // true panel
+    0..0        NonIndependentFinding
     0..0        ExceptionValue
     0..0        ReferenceRange
     0..0		AnatomicalLocation
     0..0        Specimen
     0..0        Device
-    0..1        FindingMethod from CancerStagingSystemVS
-                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
-    NonIndependentFinding
-        includes  0..1 StageGroup  // MK - should this be the FindingResult?
-    PanelMembers.Observation
-        includes  0..1 TNMClinicalPrimaryTumorClassification
-        includes  0..1 TNMClinicalRegionalNodesClassification
-        includes  0..1 TNMClinicalDistantMetastasesClassification
-        includes  0..1 TNMPathologicPrimaryTumorClassification
-        includes  0..1 TNMPathologicRegionalNodesClassification
-        includes  0..1 TNMPathologicDistantMetastasesClassification
-
-Element:        StageTiming
+    0..0        FindingMethod // The expectation is that each member of the panel will declares its own method; if added here, it would be redundant.
+
+
+EntryElement:   TNMClinicalStagePanel
+Based on:       CancerStagePanel
+Concept:        SCT#385349001   "Clinical stage (observable entity)"
```

Comment:
SCT code is incorrect. There's no "general" `CancerStage` code, only `Stage`, which is vague, in MTH. Suggest leaving as TBD.